### DISCUSSION TITLE
audio groups configuration lost

### DISCUSSION THREAD

**Discussion by KenBarta** [GrandOrgue/grandorgue#1939] [Help]
Does any know where "audio groups" are stored. It's not in the "export combinations or settings" file.  I've lost a complicated audio groups set-up that I use with an EMU 1212 card. I'm using an old GO version, but the question would apply to all versions.

**Comment by larspalo** [comment]
The audio groups are stored in the settings (.cmb) file, not in the specific combinations (.yaml) file though.

**Reply by KenBarta** [reply]
Hi Lars:  Not really, No way to retrieve or restore them. Importing the combinations file or settings file does not restore these values.

**Reply by larspalo** [reply]
No, maybe not in the way you're expecting it, or as I was assuming you were asking. But the information about to what audio group every single pipe or whatever node is/was connected to is still stored there since time immemorial. Kid me not, and don't fool anyone else about that!

In your particular case things might have changed so much that there's no easy way to port what configuration you previously had to what currently is in vouge, but that information in the .cmb is still there... The old config might not be valid nowadays though since so many things might have changed!

**Comment by KenBarta** [comment]
This is what was lost. Not in settings or cmb folder. Had to reconstruct by
hand from a photo, luckily I had.

**Comment by KenBarta** [comment]
Thanks for replying.  Ken

**Comment by larspalo** [comment]
@KenBarta Ok.  It seems to me that your particular audio configuration to specific organs couldn't be automatically restored when updating. That's unfortunate, but depending on how you were trying to restore them from your **backups** to the _updated system_, please understand that it's very hard for anyone to respond in a way that's beneficial for you unless we know all the specifics.

Still, for all purposes and situations, in the old .cmb file there would be information about to what particular audio group every pipe-config node (pipe or other element) was connected to. If that couldn't be restored in the newly updated system that's unfortunate, but it would also depend on how the update itself was performed. As like if the audio groups needed on the system were available (added by you) before attempting to "restore" them from the backup files...
